server/game/cards/01-Core/HidaKisada.js

```diff
@@ -4,7 +4,8 @@ const { Locations, CardTypes, EventNames, AbilityTypes } = require('../../Consta
 
 class HidaKisada extends DrawCard {
     setupCardAbilities() {
-        this.firstActionEvent = null;
+        this.firstActionEvent = [];
```

Comment:
This should be `this.firstActionEvent = {};`, no?

Reply:
luckily JS doesn't really care when arrays have empty slots, but yes I'll change it to an object